lib/src/constants.dart

```diff
@@ -0,0 +1,27 @@
+/// Default base URL for Eppo API
+const String precomputedBaseUrl = 'https://fs-edge-assignment.eppo.cloud';
+
+/// Precomputed flags endpoint for fetching assignments
+const String precomputedFlagsEndpoint = '/assignments';
+
+/// Default request timeout in milliseconds
+const int defaultRequestTimeoutMs = 5000;
+
+/// SDK platform enum
+enum SdkPlatform {
+  dart,
+  flutter,
+  other;
+
+  @override
+  String toString() {
+    switch (this) {
+      case SdkPlatform.dart:
+        return 'dart';
+      case SdkPlatform.flutter:
+        return 'flutter';
+      case SdkPlatform.other:
+        return 'other';
+    }
```

Comment:
I will do more research here - ideally we ask our customers (or the SDK can auto-detect) whether it is running in web, android, ios. Correctness will help our understanding of usage.